App logo

bogadeji opened the pull request:
![app logo](https://user-images.githubusercontent.com/20278394/156065942-81d998a1-9190-43a7-9b04-c006de8df2f5.PNG)
 The background color is not included. I just added that so the difference between the two logos can be seen.

bogadeji committed:
Make it possible to use absolute path for imports

bogadeji committed:
Create app logo component/atom

chuksdozie reviewed (changes requested):
You made worked on just one file, so why am I seeing changes in 7 files including package.json...
resolve this please.. I want to see only changes in the file that you worked on

bogadeji commented:
@chuksdozie in the package.json and App.js files, I had to take out merge conflicts that occurred when i pulled in from the upload branch.
2 other files are the images for the logo that I added to the repo. That's because there are two logos in the design. That's 4 files out of the 7. 
One file is the main component/atom created, that's number 5.
The 6th file was styling I added to set img width and max-width to 100%. This ensures the img does not exceed the width of the container it was put in.
The 7th file is a jsconfig file that I needed so as to remove the need for relative paths when importing local assets.

Not sure which of this files you would like me to take out.
